packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -45,6 +45,20 @@ enum ScrollbarOrientation {
   bottom,
 }
 
+/// How a scrollbar behaves in [ScrollView]s that are infinitely long.
+enum InfiniteScrollBehavior {
+  /// The scrollbar will build up additional extent as it reaches the end of the
+  /// currently assumed extent, simulating the loading of additional content.
+  page,
+
+  /// The scrollbar will always assume a constant extent ahead of it along the
+  /// scrollbar track.
+  continuous,
+
+  /// A scrollbar will not be rendered in infinite [ScrollView]s.
+  none,
```

Comment:
Why need a `none` type? Folks can remove the scrollbar widget if they do not want to show the bars, right?
Or for to not be a breaking change?

Reply:
They can remove the scrollbar from their own code, definitely. Although when the Scrollbar is being added automatically (like via the ScrollBehavior) it is a bit harder to access. This just seemed more approachable to me, especially since `none` is the current behavior, but I can remove if it's unnecessary.